frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-global.spec.ts

```diff
@@ -13,6 +13,28 @@ const operatorInstance = 'StorageCluster';
 const openshiftOperatorsNS = 'openshift-operators';
 const operandLink = 'portworx';
 
+const verifyPackageManifest = (opName: string, csName: string, depth: number = 0) => {
```

Comment:
@dtaylor113, I made an attempt at adding polling per https://github.com/openshift/console/pull/7699#discussion_r552131837.  What do you think?

If this is good, I will move this and the rest of `CatalogSource` install and uninstall out to a separate file so it can be shared between the two tests.

Reply:
:+1: I like it!